modules/quantcastIdSystem.js

```diff
@@ -34,9 +179,28 @@ export const quantcastIdSubmodule = {
    * @function
    * @returns {{id: {quantcastId: string} | undefined}}}
    */
-  getId() {
+  getId(config) {
     // Consent signals are currently checked on the server side.
     let fpa = storage.getCookie(QUANTCAST_FPA);
+
+    const coppa = coppaDataHandler.getCoppa();
```

Comment:
is coppaDataHandler always available?

Reply:
Yes, coppaDataHandler is a Prebid utility function. It checks for Coppa configuration parameter that the publisher is expected to pass. When not present, it returns undefined.